c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -4,7 +4,34 @@ This file includes migration steps for each release of c2cgeoportal.
 Version 1.6
 ===========
 
-1. Get the new and heavily modified files:
+1. Warning about .in and .mako
```

Comment:
Thanks for this informative piece of text! Capitalizing the first letter of each sentence would be more correct.